src/frontend/Player/createVideojsPlayer.ts

```diff
@@ -97,7 +97,7 @@ export const createVideojsPlayer = (
 
   const tracks = player.remoteTextTracks();
 
-  /************************** XAPI **************************/
+  /** ************************ XAPI ************************* */
```

Comment:
Maybe we could disable this? IMO most comment-related rules are garbage (see what pylint is doing on the backend in this project...)